source/options-storage.ts

```diff
@@ -19,7 +19,6 @@ const defaults = Object.assign({
 
 // TODO[2021-10-01]: Drop classes `muted-link`, `link-gray`, `link-gray-dark`, `text-gray`, `text-gray-light` and `text-gray-dark` #4021
 const migrations = [
-	featureWasRenamed('pr-easy-toggle-files', 'easy-toggle-files'), // Merged in December
```

Comment:
I think we should start leaving migrations a few more months, they don't really harm anyone. They're useful to who doesn't update super often (due to GHE or whatever reason)